cg/meta/demultiplex/housekeeper_storage_functions.py

```diff
@@ -88,9 +89,13 @@ def add_sample_fastq_files_to_housekeeper(
             continue
 
         for sample_fastq_path in sample_fastq_paths:
+            fastq_file_path_appended: Path = append_flow_cell_name_to_fastq_file_path(
```

Comment:
Same here regarding the use of appended. How about `modified_fastq_path` or `modified_fastq_file_path`?

Reply:
fixed used `renamed` pre-fix instead of `modified` to stay in line with the rest of the function body